pkg/cp/cp.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cp
+
+import (
+	"io"
+	"os"
+)
+
+// Copy src file to dst file.
+func Copy(src, dst string) error {
+	r, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer r.Close()
+
+	w, err := os.Create(dst)
+	if err != nil {
+		return err
+	}
+	defer w.Close()
```

Comment:
I keep reading that defer on open writeable files is kind of a bad idea. Since you don't have much of a complex flow here should you maybe just call w.Close directly?

Reply:
I see what you mean. I'll fix this in another PR.